controllers/templates/migrateconf.tmpl

```diff
@@ -0,0 +1,37 @@
+env {
```

Comment:
I think this file can be nicer to read if you use `with` in the right places. 

```
{{with pipeline}} T1 {{end}}
	If the value of the pipeline is empty, no output is generated;
	otherwise, dot is set to the value of the pipeline and T1 is
	executed.

```